test/test_file.py

```diff
@@ -0,0 +1,45 @@
+# This file represents all known cases of special-case indentations
```

Comment:
Good idea. I like the idea of having a "crazy scenario" file like this. I actually have one, but it's ignored in the repo :smile: